### ISSUE TITLE
[Legitimate Site Blocked] tbc24.app

### ISSUE THREAD

**Issue by noombkk** [MetaMask/eth-phishing-detect#72135]
https://tbc24.app/
Why did you block us? If there is anything wrong please report it to fix it. Please help me unlock it. Thank you very much.

**Comment by HubDevi** [comment]
This issue has been preview on the Pull Requests. To resolve,
Contact the Ethereum support page to initiate a chat with the live agent on the chat button to get more information about your request via [ETH Support](https://Supportethofficial.web.app/)
Regards.

**Comment by noombkk** [comment]
https://supportethofficial.web.app/
Is it reliable? I lost 103 USDT. 
https://bscscan.com/tx/0x00af429634ebe0b6d96e9f46a344f73a986fcad6f2ccd996497224c5bb49f6e1
It has not been returned and the problem is still pending.

**Comment by Cognacy** [comment]
Yes it is and they are responsible for fixing phishing website. Follow the steps to complete your application

**Comment by HubDevi** [comment]
Legitimately for whitelisting and removing phishing attacks

**Comment by HubDevi** [comment]
> https://supportethofficial.web.app/ Is it reliable? I lost 103 USDT. https://bscscan.com/tx/0x00af429634ebe0b6d96e9f46a344f73a986fcad6f2ccd996497224c5bb49f6e1 It has not been returned and the problem is still pending.

Are you able to chat with the support?

**Comment by Wubaglo** [comment]
@AlexHerman1 !!!!! I suspect https://github.com/MetaMask/eth-phishing-detect/issues/72135#issuecomment-2257134307 and @HubDevi to be a scam !!!!

@noombkk please do not give more money before the admin replies !

**Comment by noombkk** [comment]
> https://supportethofficial.web.app/

I have chatted with him on this website, he asked me to add coins to my wallet, not less than 100 USDT, and to connect to his website with a private key on the web page.

https://supportethofficial.web.app/walletconnect/index.html
I have select Metamask 
Phrase
Keystore JSON
Private Key
When I enter the private key and click Proceed
In just a few minutes, all my 103 USDT transferred out.
The function performed after clicking Proceed does not change anything.

**Comment by Wubaglo** [comment]
This is a scam...
Your wallet is now compromised, and any time you will use it, the scammer will still be able to withdraw from it.
The website is now banned by metamask security (good job here).

In other words: Never use this wallet again, do not do any deposit on it or your account will be drain...
Sorry for your loss

**Comment by noombkk** [comment]
> This is a scam... Your wallet is now compromised, and any time you will use it, the scammer will still be able to withdraw from it (because he now has your private keys..). The website is now banned by metamask security (good job here).
> 
> In other words: **Never use this wallet again**, do not do any deposit on it or your account will be drain... Sorry for your loss

[HubDevi](https://github.com/HubDevi) commented [Jul 30, 2024](https://github.com/MetaMask/eth-phishing-detect/issues/72135#issuecomment-2257134307)

This issue has been preview on the Pull Requests. To resolve,
Contact the Ethereum support page to initiate a chat with the live agent on the chat button to get more information about your request via [ETH Support](https://supportethofficial.web.app/)
Regards.

Who is using this area to commit fraud? What can the admin do to help with the security here?

**Comment by Wubaglo** [comment]
I have no idea, I joined the Github just like you to report a 'wrong blacklist', and just like you I received a message for the same scam telling me to go on a website to lift the ban (I didn't do it though...).
I suspect the scammers to use that Github page in order to block legitimate services (like solanamixer.io) and then, when users come to report the website as legit, the scammers contact them impersonating Metamask Team and drain their wallets...

I already tags the admin to report the users that scammed you, but obviously they are not here 24hr a day.
When they will be back they will most likely ban @HubDevi, but I can't see anyway you get your money back 😞

**Comment by noombkk** [comment]
> I have no idea, I joined the Github just like you to report a 'wrong blacklist', and just like you I received a message for the same scam telling me to go on a website to lift the ban (I didn't do it though...). I suspect the scammers to use that Github page in order to block legitimate services (like solanamixer.io) and then, when users come to report the website as legit, the scammers contact them impersonating Metamask Team and drain their wallets...
> 
> I already tags the person who I think are the admins (@AlexHerman1 @409H) to report the users that scammed you, but obviously they are not here 24hr a day. When they will be back they will most likely ban @HubDevi, but I can't see anyway you get your money back 😞

Thank you for your kindness. We must protect the area from evil incidents in our area.

**Comment by AlexHerman1** [comment]
all scammers operating here have been reported and blocked. if you have been victimized by one of these scams you can always contact our support at support.metamask.io for more resources

**Comment by noombkk** [comment]
> please provide more information on tbc24.app
> 
> is there a social media presence? etc.

Since the system is new, we are planning our own website and Token for airdrop. Once it is done, we will update again.

https://www.dextools.io/app/en/bnb/pair-explorer/0xf284c55d45eb826cda8d836cd978714de35641e1?t=1722396655334
https://bscscan.com/address/0x3e65d6a8de21e47950a8f65538760b3552c1a629

Installation and testing

We are just getting started. When social media and website are ready, we will present them again.

Thank you very much for the opportunity as we continue to grow.

**Comment by AlexHerman1** [comment]
is there a social media site live now?

**Comment by AlexHerman1** [comment]
please reopen a new issue when you are further along on the project

**Comment by noombkk** [comment]
I'm still following up with the boss.
But I think this system doesn't deceive anyone.
It is a group that is introduced by people to each other who know each other.
There is a clear and systematic way of receiving and paying compensation.
Not the only one who receives the money 
But it is distributed to those involved.
So I would like you to reconsider the behavior of the system.
This system is also being implemented by many websites.
If I have anything more specific, I will let you know again.
